app/views/showApplication.scala.html

```diff
@@ -377,13 +383,12 @@ <h2 class="mdl-card__title-text">@application.subject</h2>
                                 </ul>
                             }
                             <p class="application__message">@answer.message</p>
-                            @if(application.answers.last == answer && application.creatorUserId == currentUser.id && !application.closed && answer.creatorUserID != currentUser.id) {
+                            @if(answer.answerType === AnswerType.ApplicationProcessed && application.creatorUserId == currentUser.id && not(application.closed) && answer.creatorUserID != currentUser.id) {
                                 <div class="info-box do-not-print">
-                                Si l’instructeur a répondu à votre demande, vous pouvez archiver l’échange :<br><br>
-                                <button class="mdl-button mdl-js-button mdl-button--raised mdl-button--primary mdl-js-ripple-effect" onclick="showDialog()" >
+                                    Cette demande a bien été traitée. Je vous invite à archiver l'échange en cliquant sur le bouton ci-dessous :<br><br>
```

Comment:
```suggestion
                                    Cette demande a bien été traitée. Je vous invite à archiver l’échange en cliquant sur le bouton ci-dessous&nbsp;:<br><br>
```
apostrophe courbe et espace insécable U+00A0 (j'ai mis `&nbsp;` parce que j'arrive pas à ajouter le bon char dans l'interface github)